Resources/views/PlatformUI/shell.html.twig

```diff
@@ -80,6 +80,9 @@
                             "fieldViews": {
                                 "ezcountry": {{countriesInfo|json_encode|raw}}
                             }
+                        },
+                        "editContent": {
+                            "editViews": {}
```

Comment:
to be consistent with the field view case, it should be `fieldEditViews` instead of the `editViews`